### DISCUSSION TITLE
How to enter fullscreen?

### DISCUSSION THREAD

**Discussion by UlyssesZh** [Genymobile/feedback#45] [Genymotion Desktop] [answered]
I have just upgraded to v3.3. Previously I can use <kbd>F11</kbd> to enter fullscreen. Now I noticed that the shorcut changed to <kbd>Alt</kbd> + <kbd>F11</kbd>. However, now the fullscreen mode does not hide the tool bar. How can I enter a fullscreen mode with the tool bar hidden?

**Comment by agavignet** [comment, marked as answer]
Hi @UlyssesZh

You can now customize shortcuts (see https://docs.genymotion.com/desktop/04_Emulating_sensors_and_features/#shortcuts) to fit your need.
Make sure to use the latest Desktop version 3.3.2 because the behavior you describe might have been fixed (see release notes https://www.genymotion.com/blog/release-note/genymotion-desktop-3-3-2/).
You can still hide the toolbar afterward with Ctrl+T.

Best regards.
